src/main/java/uk/co/ramp/covid/simulation/place/Restaurant.java

```diff
@@ -15,22 +15,25 @@ public Restaurant() {
     public Restaurant(Size s) {
         super(s);
         transProb = PopulationParameters.get().getpBaseTrans() * PopulationParameters.get().getpRestaurantTrans();
-        startDay = 1;
-        endDay = 7;
-        startTime = 10;
-        endTime = 22;
+        times.setOpen(10);
```

Comment:
Should we make restaurant open and close times variable. Not all restaurants open at 10am

Reply:
Paul has suggested a set of times (based on shop sizes). These are coming in a future commit!